contrib/seccomp/seccomp.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
+
+package seccomp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/containers"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// WithProfile sets the provided seccomp profile to the spec.
```

Comment:
Maybe specify that this is a file on disk in json format

Reply:
done..